b2share/modules/schemas/errors.py

```diff
@@ -82,6 +82,38 @@ class BlockSchemaVersionIsReleased(B2ShareSchemasError):
     """Exception raised while trying to modify a released block schema."""
     pass
 
+
+#
+# BLOCK SCHEMA VERSION ERRORS
+#
+
+class InvalidSchemaVersionError(B2ShareSchemasError):
+    """Exception raised when trying to add a version with wrong id."""
+
+    MESSAGE = """Version number is invalid. Provided version number should
+                follow the last existing version number, which currently
+                is {0}."""
+
+    def __init__(self, schema_number):
+        self.schema_number = schema_number
+        super(InvalidSchemaVersionError, self).__init__(
+            self.MESSAGE.format(schema_number)
+        )
+
+
+class SchemaVersionExistsError(B2ShareSchemasError):
```

Comment:
The two exception class names look a little strange. Exception `SchemaVersionExistsError` is raised when the version is invalid. Thus it is kind of strange that it does not inherit the existing exception `InvalidSchemaVersionError `.
What about having a generic exception `InvalidSchemaVersionError` and an additional `SchemaVersionSkippedError`? Both `SchemaVersionSkippedError` and `SchemaVersionExistsError ` would inherit `InvalidSchemaVersionError`.